AK/CircularQueue.h

```diff
@@ -55,7 +55,6 @@ class CircularQueue {
         VERIFY(!is_empty());
         auto& slot = elements()[m_head];
         T value = move(slot);
-        slot.~T();
```

Comment:
This change is incorrect; the destructors of moved-from objects are run regardless according to C++ semantics (e.g. in the case of variables on the stack).

Reply:
well `clang-sa` pointed this one out, but I see your point
not sure if I should mark this somehow (launder?), but I will drop this commit for now